sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/RedundantArgumentTest.cs

```diff
@@ -31,40 +31,34 @@ public class RedundantArgumentTest
     {
         [TestMethod]
         [TestCategory("Rule")]
-        public void RedundantArgument()
-        {
+        public void RedundantArgument() =>
             Verifier.VerifyAnalyzer(@"TestCases\RedundantArgument.cs",
-                new RedundantArgument(),
-                options: ParseOptionsHelper.FromCSharp8,
-                additionalReferences: NuGetMetadataReference.NETStandardV2_1_0);
-        }
+                                    new RedundantArgument(),
+                                    ParseOptionsHelper.FromCSharp8,
+                                    additionalReferences: NuGetMetadataReference.NETStandardV2_1_0);
 
+#if NETFRAMEWORK // ToDo: Fix this when running on .Net Core
```

Comment:
what's failing here?

it seems inconsistent that 
- sometimes, you add a issue link
- sometimes, the comment is that an API doesn't work
- here, the comment says to fix

Reply:
Yes, the answer is simple, it's something I wanted to address and forgot about it.

Reply:
In the end I've created a new issue: https://github.com/SonarSource/sonar-dotnet/issues/3430.
It seems that on .Net Core 3.1 the generated code is incorrect and will not compile.

Reply:
it's a good issue to monitor in the backlog!